### PR TITLE
OCLOMRS-221 : Rework the Back to page links to be consistent across the app

### PR DESCRIPTION
**Pull request by alexmochu**
# JIRA TICKET NAME:
[Rework the Back to page links to be consistent across the app](https://issues.openmrs.org/browse/OCLOMRS-221)

# Summary:
Currently, the Back to page links UI is not consistent across the app.
Therefore the Back to page links UI should be made consistent across the app.

### CODE REVIEW

File: src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx [hunk 1]
```diff
@@ -114,23 +125,6 @@ const DictionaryDetailCard = (props) => {
               Go to concepts
             </Link>
           </fieldset>
-          <fieldset>
```
Comment (kmunene) [inline src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: Why was this deleted? Was it part of the issue scope?

Comment (alexmochu) [reply src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: @kmunene Thank you for making that observation. No, it was not part of this issue scope. As agreed by the team, I have removed those changes that were fixing a couple of bugs and I have created a ticket to specifically handle that here: https://issues.openmrs.org/browse/OCLOMRS-223
